# Series 06
## Aufgabe 6.1
In einem Land der Dritten Welt leiden 1 % der Menschen an einer bestimmten Infektionskrankheit. Ein Test zeigt die Krankheit bei den tatsächlich Erkrankten zu 98 % korrekt an. Leider zeigt der Test auch 3 % der Gesunden als erkrankt an.  
Wir bezeichnen mit K eine kranke Person und T eine positiv getestete Person.

a) Interpretieren (nicht berechnen!) Sie die Wahrscheinlichkeiten:  
$P(K)$,$P(\overline{T})$,$P(K|T)$,$P(K)$,$P(\overline{T}|\overline{K})$

* $P(K)$: Wahrscheinlichkeit an einer Infektionskrankheit zu leiden
* $P(\overline{T})$: eine negativ getestete Person
* $P(K|T)$: Wahrscheinlichkeit, dass eine Person unter der Voraussetzung eines positiven Tests krank ist.
* $P(T|K)$: Wahrscheinlichkeit, dass ein Test positiv ist, unter der Voraussetzung, dass der Mensch krank ist.
* $P(\overline{T}|\overline{K})$: Wahrscheinlichkeit, dass ein Test negativ ist, unter der Voraussetzung dass der Mensch gesund ist.

b) Bezeichnen Sie die Wahrscheinlichkeit, die in der Aufgabe gegeben sind, wie in a)

* 1% der Menschen in einem Drittweltland leiden an einer Infektionskrankheit $\to$ $P(K)$
* Der Test zeigt an, dass die Person krank ist $\to P(K|T) = 98%$
* Bei 3% der Gesunden fällt der Test positiv aus $\to P(T|\overline{K})$

c) Berechnen Sie $P(\overline{K})$

$P(\overline{K})$

In [13]:
K <- 0.01
TK <- 0.98
TnK <- 0.03
nK <- 1-K
nK

[1] 0.99

d) Mit welcher Wahrscheinlichkeit zeigt der Test bei einer zufällig ausgewählten Person ein positives Ergebnis?  
Verwenden Sie das Gesetz der totalen Wahrscheinlichkeit.

$P(T)=P(T|K)*P(K)+P(T|\overline{K})*P(\overline{K})$

In [14]:
T <- TK*K+TnK*nK
T

[1] 0.0395

e) Mit welcher Wahrscheinlichkeit ist eine positiv getestete Person auch tatsächlich krank?  Kommentieren Sie das Ergebnis.  
Verwenden Sie das Bayes Theorem.

$P(K|T) = \frac{P(T|K)*P(K)}{P(T)}$

In [15]:
KT <- TK*K/T
KT

[1] 0.2481013

Das bedeutet dass 24.8% aller positiv Getesteten auch effektiv krank ist.

f) Mit welcher Wahrscheinlichkeit ist eine als negativ getestete Person gesund? Kommentieren Sie das Ergebnis.  
Verwenden Sie das Bayes Theorem.

In [19]:
nKnT <- (1-TnK)*nK/(1-T)
nKnT

[1] 0.9997918

## Aufgabe 6.2
Bei einer Sportveranstaltung wird ein Dopingtest durchgeführt. Wenn ein Sportler gedopt hat, dann fällt der Test zu 99 % positiv aus.  
Hat ein Sportler aber nicht gedopt, zeigt der Test trotzdem zu 5 % ein positives Ergebnis an. Aus Erfahrung weiss man das 20 % der Sportler gedopt sind.  

In [24]:
TD <- 0.99
D <- 0.2
TnD <- 0.05
nD <- 1-D

a) Wie gross ist die Wahrscheinlichkeit, dass eine Dopingprobe positiv ausfällt.  
$P(T)=P(TD|D)*P(D)+P(TnD|nD)*P(nD)$

In [30]:
T <- TD*D+TnD*nD
T

[1] 0.238

b) Wie gross ist die Wahrscheinlichkeit, dass der Test negativ ausfällt, obwohl der Sportler gedopt hat?

In [31]:
nTD <- 1-TD
nTD

[1] 0.01

c) Wie gross ist die Wahrscheinlichkeit, dass ein Sportler gedopt hat, falls seine Dopingprobe negativ ausgefallen ist.  
$P(D|nT)=\frac{P(nT|D)*P(D)}{P(nT)}$

In [33]:
DnT <- nTD*D/(1-T)
DnT

[1] 0.002624672

## Aufgabe 6.3
Ein Lügendetektortest wird routinemässig bei Angestellten durchgeführt, die in sensitiven Positionen arbeiten. Bezeichnen wir nun mit $+$ das Ereignis, dass der Test positiv ist, also dass der Lügendetektor angibt, dass der Angestellte gelogen habe. Mit $W$ wird das Ereignis bezeichnet, dass der Angestellte die Wahrheit sagte und mit $L$, dass der Angestellte gelogen hat.  
Aus Untersuchungen über Lügendetektoren wissen wir, dass
* $P(+|L) = 0.88$
* $P(-|W) = 0.86$
* $P(W) = 0.99$   

a) Interpretieren Sie die Wahrscheinlichkeiten $P(W)$ und $P(+|L)$.

In [34]:
TL <- 0.88
nTW <- 0.86
W <- 0.99

$P(W)$ bedeutet, dass 99% aller Angestellten die Wahrheit sagen. 
$P(+|L)$ ist die Wahrscheinlichkeit, dass der Detektor die Lüge entdeckt, wenn der Angestellte gelogen hat

b) Bei einer Person zeigt der Detektor an, dass gelogen wurde. Mit welcher Wahrscheinlichkeit hat diese Person effektiv gelogen?  

$P(L|+)=\frac{P(+|L)P(L)}{P(+|L)P(L)+P(+|W)P(W)}$

In [37]:
LT <- TL*(1-W)/(TL*(1-W)+(1-nTW)*W)
LT

[1] 0.05970149

c) Interpretieren Sie das Resultat aus b) in 2-3 Sätzen. Für wie aussagekräftig halten Sie den Lügendetektor.

Der Lügendetektor ist unzuverlässig. Lediglich 5% der Lügen kann er erkennen.

## Aufgabe 6.4
Der Serumtest untersucht schwangere Frauen auf Babys mit Down-Syndrom. Der Test ist ein sehr guter, aber nicht perfekter Test. Ungefähr 1 % der Babys haben das Down-Syndrom. Wenn das Baby das Down-Syndrom hat, besteht eine 90-prozentige Wahrscheinlichkeit, dass das Ergebnis positiv ausfällt. Wenn das Baby nicht betroffen ist, besteht immer noch eine 1 % Chance, dass das Ergebnis positiv ist. Eine schwangere Frau wurde getestet und das Ergebnis ist positiv. Wie gross ist die Wahrscheinlichkeit, dass ihr Baby tatsächlich das Down-Syndrom hat?

In [1]:
D <- 0.01
TD <- 0.9
TnD <- 0.01

[1] 0.9889909

$P(D|T)=\frac{P(T|D)*P(D)}{P(T)}$

In [41]:
DT <- TD*D/((TD*D)+TnD*(1-D))
DT

[1] 0.4761905

## Aufgabe 6.5
Die Rauchsensoren in einer Fabrik melden ein Feuer mit Wahrscheinlichkeit 0.95. An einem Tag ohne Brand geben sie mit Wahrscheinlichkeit 0.01 falschen Alarm. Pro Jahr rechnet man mit einem Brand.  

In [46]:
F <- 1/365
AF <- 0.95
AnF <- 0.01

a) Die Alarmanlage meldet Feuer. Mit welcher Wahrscheinlichkeit brennt es tatsächlich?

In [54]:
A <- (AF*F+AnF*(1-F))
FA <- AF*F/A
FA

[1] 0.2069717

b) In einer Nacht ist es ruhig (kein Alarm). Mit welcher Wahrscheinlichkeit brennt es tatsächlich nicht?

In [55]:
nF <- (1-F)*(1-AnF)/(1-A)
nF

[1] 0.9998613